x-pack/plugins/osquery/public/packs/form/shards/shards_form.tsx

```diff
@@ -43,6 +39,13 @@ const ShardsFormComponent = ({
     }
   }, [index, onDelete]);
 
+  const buttonWrapperCss = useCallback(
+    () => css`
+      margin-top: ${index === 0 && '16px'};
+    `,
+    [index]
+  );
```

Comment:
Thoughts about converting this to a CSS obj instead? Also, you can use an EUI var here instead of `16px`:
```suggestion
  const buttonWrapperCss = useCallback(
    ({ euiTheme }) => (index === 0 ? { marginTop: euiTheme.size.base } : {}),
    [index]
  );
```

Reply:
I don't remember correctly, but I think I had some issues with passing a theme here in other way. I'll check again ;)